packages/node/src/constants.ts

```diff
@@ -21,4 +21,5 @@ export const DEFAULT_OPTIONS: Options = {
   transportClass: null,
   // By default, events flush on the next event loop
   uploadIntervalInSec: 0,
+  minIdLength: null,
```

Comment:
Should default [`minIdLength`](https://developers.amplitude.com/docs/http-api-v2#properties-2) be 5?

Reply:
I say no so that the data pipeline team can change this w/o us having to make any code changes and avoid versioning issues